garage/envs/normalized_env.py

```diff
@@ -106,5 +106,8 @@ def step(self, action):
     def log_diagnostics(self, paths):
         pass
 
+    def render(self, *args, **kwargs):
```

Comment:
Does this render function need the `mode` parameter? Just curious.

Reply:
I didn't think it over. The current render works. Should I overrides render in gym.Wrapper?
```
#gym.Wrapper
    def render(self, mode='human'):
        return self.env.render(mode)
```

Reply:
Either is acceptable. This version is more immune to refactoring and API changes because it passes all args and kwargs. It just says "call the render function on self.env with exactly the same args).